### DISCUSSION TITLE
Why not delete only disable SOC accounts?

### DISCUSSION THREAD

**Discussion by tomWhilbert** [Security-Onion-Solutions/securityonion#3065] [Unsupported Versions] [answered]
The documentation shows how to disable but not delete a SOC account.  Wondering  why that is or if there may be a way?

Thanks

**Comment by tomWhilbert** [comment]
Not seeing my post...

**Comment by weslambert** [comment, marked as answer]
Many orgs might want to audit that an account existed, but was disabled. Also, there's not a single way to delete from all tools at the moment.  You can do something like `so-user delete user@email.address `, but this will only delete the account from SOC, and not TheHive or Fleet (but these accounts will get disabled).
